src/ARToolkitNFT.ts

```diff
@@ -33,13 +33,13 @@ export default class ARToolkitNFT {
     }).then((resolve) => {
       console.log(resolve);
       
-      //this.instance = resolve.instance;
+      this.instance = resolve;
     })
     //this.instance = runtime.instance
     this._decorate()
 
     let scope = (typeof window !== 'undefined') ? window : global
-    //scope.artoolkitNFT = thisS
+    //scope.artoolkitNFT = this
```

Comment:
this is not allowed with typescript i get this error: 
```
Property 'artoolkitNFT' does not exist on type '(Window & typeof globalThis) | (Global & typeof globalThis)'.
  Property 'artoolkitNFT' does not exist on type 'Window & typeof globalThis'.ts(2339)
```
How to fix this?

Reply:
done with the global object declaration see https://github.com/webarkit/jsartoolkitNFT/pull/89/commits/f3d76587519ab85c44653fbf24c406f0914db0d2